interface/main/dated_reminders/dated_reminders_add.php

```diff
@@ -59,6 +59,9 @@
 // default values for $this_message    
     $this_message = array('message'=>'','message_priority'=>3,'dueDate'=>'');
     $forwarding = false;
+
+// default values for Max words to input in a reminder
```

Comment:
You could consider -
1. Like we did per [this post](https://sourceforge.net/p/openemr/discussion/202506/thread/6f3052be/?limit=25#3d11), fix database - 255 is as arbitrary as 160!  No changes to the existing code.    OR
2. Modify textarea as below  OR
3. Your approach is as good as any.

`<textarea maxlength="160" onKeyUp="limitText(this.form.message,this.form.countdown,160);"`

There should be no need for onKeyDown.  Also fix initial value - unless you want to let jQuery initialize it.

`<input style="border:0; background:none;" readonly type="text" name="countdown" size="3" value="160"> `